velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -675,6 +676,21 @@ std::vector<core::TypedExprPtr> ExpressionFuzzer::generateLikeArgs(
   return inputExpressions;
 }
 
+// Specialization for the "split" function.
+// Spark implementation only takes 2 args, and requires the delimiter to
```

Comment:
This is generic Fuzzer code, but it has logic that's specific to Spark (and Presto) functions. That's not ideal. Let's move this logic to specific fuzzers or use the fix from #4140